### PR TITLE
chore: create standardized repo template

### PR DESCRIPTION
**Pull request by Racer159**
This PR updates this repo to a standard that will be applied to all Software Factory repos:

https://github.com/defenseunicorns/uds-package-mattermost
https://github.com/defenseunicorns/uds-package-gitlab-runner
https://github.com/defenseunicorns/uds-package-gitlab

These changes are also dependent on the workflow naming changes in: 

https://github.com/defenseunicorns/uds-package-sonarqube/pull/28

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -76,32 +70,23 @@
...
     {
-      "fileMatch": ["^Makefile$"],
+      "fileMatch": [
+        "(^|/)zarf.yaml$"
+      ],
+      "matchStringsStrategy": "recursive",
       "matchStrings": [
-        "renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?( extractVersion=(?<extractVersion>.*?))?( registryUrl=(?<registryUrl>.*?))?\\s.*?=\\s*['\"]?(?<currentValue>.*?)['\"]?\\s"
+        "charts:(.|\n)*?(^\\s{4}[\\w:]+|\n$|^\\s{2}-)",
+        "- name: (?<depName>.+)(.|\n)*?url: (?<registryUrl>.+)(.|\n)*?version: (?<currentValue>.+)",
```
Comment (zachariahmiller) [inline renovate.json:86]: I think this doesn't capture all variations of how the component could be defined, but happy to be wrong. For example "namespace" could come after name. see [here](https://github.com/defenseunicorns/uds-core/blob/5e960c0479e6fc96244db0230296c94e936e57d8/renovate.json#L79)

**Review by mjnagel — COMMENTED** [review]
LGTM overall, good improvements to docs/templates/renovate.

File: renovate.json (hunk 1 above)
Comment (mjnagel) [reply renovate.json:86]: `namespace` gets globbed up in the `(.|\n)*?` before `url`, if present I believe, so I think this will handle all the things.

Comment (Racer159) [reply renovate.json]: You passed your first test @zachariahmiller - never trust me - here is a test showing the update works though: https://github.com/Racer159/uds-package-sonarqube/pull/3 (I did have to remove the encrypted username/ password from that but the regexManager is the same)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,12 +16,42 @@ Specifically:
...
+2. :key: Set up your Git config to GPG sign all commits. [Here's some documentation on how to set it up](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). You won't be able to merge your PR if you have any unverified commits.
+3. Create a Draft Pull Request as soon as you can, even if it is just 5 minutes after you started working on it. We lean towards working in the open as much as we can.
+   > ⚠️ **NOTE:** _:key: We use [Conventional Commit messages](https://www.conventionalcommits.org/) in PR descriptions so, if you can, use one of `fix:`, `feat:`, `chore:`, `docs:` or similar.  If you need help, just use with `wip:` and we'll help with the rest_
+4. :key: Automated tests will begin based on the paths you have edited in your Pull Request.
+   > ⚠️ **NOTE:** _If you are an external third-party contributor, the pipelines won't run until a [CODEOWNER](./CODEOWNERS) approves the pipeline run._
+5. :key: Be sure to use the `needs-adr`,`needs-docs`,`needs-tests` labels as appropriate for the PR. Once you have addressed all of the needs, remove the label.
```
[the remark this answers is not in the document]
Comment (Racer159) [reply CONTRIBUTING.md]: fixed! lemme know if there are more tweaks

**Review by mjnagel — APPROVED** [review]

**Review by zachariahmiller — APPROVED** [review]

**Pull request merged**